features/frameworks/greentea-client/greentea-client/test_env.h

```diff
@@ -28,6 +28,9 @@
 #endif
 
 #include <stdio.h>
+extern "C" {
```

Comment:
OK. Actually that's the point of splitting into two files. The ```test_env.h``` is included only by C++ code hence does not need ```ifdef __cplusplus```. Similarly ```test_enc_c.h``` is only included in the C code (as suggested by '_c') hence no need to put ```extern "C"``` there.